packages/taquito/test/helpers.ts

```diff
@@ -591,3 +591,89 @@ export const ticketTokenTestMock = {
     ],
   },
 };
+
+export const smallerTypeCheckError = {
```

Comment:
might want to rename this to something a bit more descriptive, it isn't clear what a smaller typecheck error is